libraries/C6502Cpu/CGravitarGame.cpp

```diff
@@ -0,0 +1,178 @@
+//
+// Copyright (c) 2021, Marc Deslauriers
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright notice,
+//    this list of conditions and the following disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright notice,
+//    this list of conditions and the following disclaimer in the documentation
+//    and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS
+// OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+// COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+// TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+// EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+#include "CGravitarGame.h"
+
+//
+// Gravitar Set 1
+//
+//                                         01   02   04   08   10   20   40   80  100  200  400  800
+static const UINT16 s_romData2n_1_1[] = {0x16,0x85,0x85,0x85,0x85,0x85,0x7a,0x57,0xab,0x20,0xbb,0x00}; // 136010-101.d1
```

Comment:
You're adding a lot of image sets here. The way the Arduino compiles these means they end up in data RAM and there is only 8Kb of that. Games that I've been touching I've been moving over to a different way of expressing data2n - see CGalaxianGame.cpp for a space saving format and invocation using a new CGame constructor.

Reply:
Oh, cool. Let me fix that up and I'll update the PR.